mode/examples/Demos/Graphics/BoxClock/BoxClock.pyde

```diff
@@ -51,7 +48,9 @@ def drawShape():
 
 
 def makeEdgeCube():
-    # Draw a 2x2x2 transparent box with different-colored edges.
+    global edgeCube
```

Comment:
and then return edgeCube from this rather than assigning to it here.

Then the global decls are in setup where I think they're easier to keep track of and less magical.